images/ceph-toolbox/Makefile

```diff
@@ -46,10 +46,4 @@ do.build: toolbox-base-image
 	@cp -a . $(TEMP)
 	@cd $(TEMP) && $(SED_CMD) 's|BASEIMAGE|$(TOOLBOX_BASE_IMAGE)|g' Dockerfile
 	@docker build $(BUILD_ARGS) -t $(TOOLBOX_IMAGE) $(TEMP)
-	@[ "$(OLD_IMAGE_ID)" != "$(CURRENT_IMAGE_ID)" ] && [ -n "$(OLD_IMAGE_ID)" ] && docker rmi $(OLD_IMAGE_ID) || true
```

Comment:
I think you can remove OLD_IMAGE_ID and CURRENT_IMAGE_ID above too.

Reply:
Good catch. My most recent push should fix all these.